source/fundamentals/aggregation/aggregation-expression-operators.txt

```diff
@@ -0,0 +1,32 @@
+=================================
+Aggregation Expression Operations
+=================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to construct expressions 
+for use in the aggregation pipeline. 
+With the {+driver-short+}, you can perform expression operations with
+discoverable, typesafe Java methods rather than BSON documents.
+Because these methods follow the fluent interface pattern, you can chain 
+aggregation operations together to create code that is both more compact and 
+more naturally readable.  
+
+.. tip:: 
+
+   For more information about MQL aggregation expressions, see the 
+   :manual:`Aggregation Pipeline Operators <https://www.mongodb.com/docs/manual/reference/operator/aggregation/>` 
```

Comment:
Link appears to be broken on staging?

Reply:
D'oh. Used the full link instead of starting with `/reference`